webClient/src/app/activity-inprogress-table/activity-inprogress-tablel.component.ts

```diff
@@ -0,0 +1,184 @@
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+
+import { Component, EventEmitter, Output, Inject, Input,ViewChild,TemplateRef } from '@angular/core';
+import { Connection } from '../Connection';
+import { FTAActivityService } from '../services/FTAActivity.service';
+import { FTAActivity } from '../services/FTAActivity.service';
+import { DownloadService } from '../services/Download.service';
+import { TableItem, TableHeaderItem, TableModel,TableExpandedRow} from 'carbon-components-angular/table/table.module';
+
+import { SelectItem } from 'primeng/api';
+import { Message } from 'primeng/components/common/api';
+import { Angular2InjectionTokens,Angular2PluginViewportEvents } from 'pluginlib/inject-resources';
+import * as globals from '../../environments/environment';
+
+@Component({
+  selector: 'activity-progress-table',
+  templateUrl: './activity-inprogress-tablel.component.html',
```

Comment:
"tablel" should be "table" 😃 in a few places here

Reply:
fixed.